From fc1708d81abaa933c163587ecb48dd246d86a35b Mon Sep 17 00:00:00 2001
From: Tomas Kopecek <tkopecek@redhat.com>
Date: Tue, 7 Dec 2021 11:28:03 +0100
Subject: [PATCH] Return mistakenly dropped option (--keytab)

PR https://pagure.io/koji/pull-request/3051 was wrongly merged
https://pagure.io/koji/c/071533206e6d29f665d7a5864eda75087488a3ce?branch=master
and deleted wrong line. Reverting it.

Fixes: https://pagure.io/koji/issue/3171
---
 cli/koji | 1 +
 1 file changed, 1 insertion(+)

diff --git a/cli/koji b/cli/koji
index a0a9bd7f..87fd6f15 100755
--- a/koji
+++ b/koji
@@ -120,6 +120,7 @@ def get_options():
     parser.add_option("-c", "--config", dest="configFile",
                       help="load profile's settings from another file. Use with --profile.",
                       metavar="FILE")
+    parser.add_option("--keytab", help="specify a Kerberos keytab to use", metavar="FILE")
     parser.add_option("--principal", help="specify a Kerberos principal to use")
     parser.add_option("--cert", help="specify a SSL cert to use", metavar="FILE")
     parser.add_option("--runas", help="run as the specified user (requires special privileges)")
-- 
2.33.1

